packages/app/ios/ShenaniganApp/Info.plist

```diff
@@ -39,12 +39,12 @@
 	</dict>
 	<key>NSCameraUsageDescription</key>
 	<string>Shenanigan requires access to your phone’s camera.</string>
-	<key>NSPhotoLibraryUsageDescription</key>
```

Comment:
Not sure why this is changed?

Reply:
I was unsure about this as well but saw that this file had been committed. So i went ahead. Should I revert this change?

Reply:
Should I revert this change?